cypress/integration/app/localStorage.spec.js

```diff
@@ -0,0 +1,27 @@
+/// <reference types="cypress" />
+
+context('Local Storage', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000')
+  })
+
+  it('should write and clear localStorage', () => {
+    cy.get('[name="projectName"]').type('Summer dress with sleeves')
+    cy.get('[name="pattern"]').type('Pinterest')
+    cy.get('[name="size"]').type('38')
+    cy.get('[name="nextStep"]').type(
+      'cut fabric and lining, buy button, zipper and tape'
+    )
+    cy.get('button')
+      .click()
+      .should(() => {
+        expect(localStorage.getItem('projects')).to.eq(
+          '[{"projectName":"Summer dress with sleeves","pattern":"Pinterest","size":"38","nextStep":"cut fabric and lining, buy button, zipper and tape"}]'
+        )
+      })
+
+    cy.clearLocalStorage().should((localStorage) => {
```

Comment:
Hast du eine bestimmte Bedingung, wann der Local Storage geleert werden soll? Also könnte der Nutzer das von selber machen? Oder wolltest du nur testen, ob es, wenn es leer ist auch nichts zurückgibt?

Reply:
Ich hatte das in dem example zum local Storage Test von cypress gesehen, und dachte es wäre ja garnicht so schlecht das mitzutesten & außerdem schreibt er mir dann nicht meinen local Storage voll bzw. ich muss ihn nicht händisch löschen. Weiß nicht genau ob Cypress das nicht sowieso von allein macht hinterher ohne diesen Zusatz, aber unsere Datenbank hatte es uns damals ja auch vollgeschrieben...